infra/nnfw/cmake/buildtool/cross/toolchain_aarch64-linux.cmake

```diff
@@ -1,20 +1,27 @@
 #
 # config for aarch64-linux
 #
-include(CMakeForceCompiler)
 
+# Set CMAKE_SYSTEM_NAME to notify to cmake that we are cross compiling
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR aarch64)
 
-set(CMAKE_C_COMPILER   aarch64-linux-gnu-gcc)
-set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
-
-# where is the target environment
-set(NNAS_PROJECT_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../../..")
-set(ROOTFS_AARCH64 "${NNAS_PROJECT_SOURCE_DIR}/tools/cross/rootfs/aarch64")
-include("${NNAS_PROJECT_SOURCE_DIR}/infra/cmake/modules/OptionTools.cmake")
+if(DEFINED ENV{USE_CLANG} AND "$ENV{USE_CLANG}" STREQUAL "1")
```

Comment:
just a question, 
why did you add clang support?

Reply:
- Sometimes someone request clang build (ex. oversea member)
- More strict frontend check than gcc, and gcc follows these check on version up